gitops_cli_argocd/argocd-gitops-cli-reference.adoc

```diff
@@ -29,5 +29,5 @@ include::modules/gitops-argocd-cli-utility-commands.adoc[leveloffset=+1]
 * xref:../gitops_cli_argocd/configuring-argocd-gitops-cli.adoc#configuring-argocd-gitops-cli[Configuring the {gitops-shortname} CLI]
 * xref:../gitops_cli_argocd/logging-in-to-argocd-server-in-default-mode.adoc#logging-in-to-argocd-server-in-default-mode[Logging in to the Argo CD server in the default mode]
 * link:https://github.com/redhat-developer/gitops-operator/blob/7ac4b2ce179c167b39be259b8d9be37dc280f689/docs/OpenShift%20GitOps%20CLI%20User%20Guide.md#login-related-commands[OpenShift {gitops-shortname} CLI User Guide]
-* link:https://argo-cd.readthedocs.io/en/release-{upstream-ver}/user-guide/commands/argocd/[`argocd` command reference]
+* link:https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd/[`argocd` command reference]
```

Comment:
```suggestion
* link:https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd/[`argocd` Command Reference]
```
To match with the title in the page that this leads to.

Reply:
Fixed.